admin_tests/base/test_utils.py

```diff
@@ -158,3 +165,48 @@ def test_change_embargo_date(self):
         assert_almost_equal(self.registration.embargo.end_date, self.date_valid2, delta=datetime.timedelta(days=1))
 
         # Add a test to check privatizing
+
+site = AdminSite()
+
+class TestGroupCollectionsPreprints(AdminTestCase):
+
+    @pytest.mark.enable_bookmark_creation
+    def test_admin_app_groups(self):
+        user = UserFactory()
+        user.save()
+        request = RequestFactory()
+        get_request = request.get('/admin/osf/osfuser/{}/change'.format(user.id))
+        get_request.user = user
+        post_request = request.post('/admin/osf/osfuser/{}/change'.format(user.id))
+        post_request.user = user
+        preprint = PreprintFactory(creator=user)
+        preprint.save()
+        admin = OSFUserAdmin(OSFUser, site)
+        formfield = (admin.formfield_for_manytomany(OSFUser.groups.field, request=get_request))
+        queryset = formfield.queryset
+
+        collections_group = Group.objects.filter(name__startswith='collections_')[0]
+        assert(collections_group not in queryset)
+
+        preprint_group = Group.objects.filter(name__startswith='preprint_')[0]
+        assert(preprint_group not in queryset)
```

Comment:
Can you explain to me what this assertion if for? I don't follow what you're doing with the `get_request` part of the test.  Also, an additional comment would be helpful.